jetbrains-core/src/software/aws/toolkits/jetbrains/ui/HandlerPanel.kt

```diff
@@ -59,8 +59,8 @@ class HandlerPanel(private val project: Project) : JPanel(MigLayout("novisualpad
         switchCompletion()
     }
 
-    fun setRuntime(runtime: Runtime) {
-        this.runtime = runtime
+    fun setRuntime(runtime: Runtime?) {
+        this.runtime = runtime ?: Runtime.UNKNOWN_TO_SDK_VERSION
```

Comment:
It works either way, but why are we using `UNKNOWN_TO_SDK_VERSION` directly instead of making it nullable?

Reply:
Changed